### PR TITLE
Module3_Task1

### PR DESCRIPTION
**Pull request by OlegMazurenko**

### CODE REVIEW

File: Module3/Models/Product.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DeliveryService.Models
+{
+    public class Product : BaseModel
+    {
+        public string Name { get; set; }
```
Comment (yevhenbilushko) [inline Module3/Models/Product.cs:9]: seems like it should be OrderId

File: Module3/Models/Order.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    public class Order : BaseModel
+    {
+        public int UserId { get; set; }
+        public string Address { get; set; }
+        public IList<Product> Products { get; set; }
+        public decimal DeliveryCost { get; set; }
+        public decimal TotalPrice { get; set; }
+        public OrderStatus Status { get; set; }
+
+        public Order(string address, IList<Product> products)
```
Comment (yevhenbilushko) [inline Module3/Models/Order.cs]: no need method
it is only for DB structure

File: Module3/Models/Product.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        public string Description { get; set; }
+        public decimal Price { get; set; }
+        public int SellerId { get; set; }
+        public int CategoryId { get; set; }
+
+        public Product(string name, string description, decimal price)
```
Comment (yevhenbilushko) [inline Module3/Models/Product.cs]: no need

File: Module3/Models/User.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace DeliveryService.Models
+{
+    public class User : BaseModel
+    {
+        public string PhoneNumber { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public string Name { get; set; }
+        public AccessLevel Access { get; set; }
```
Comment (yevhenbilushko) [inline Module3/Models/User.cs:13]: seems like u can add list of user orders

File: Module3/Models/User.cs [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+        public User(string email, string password, string name, string phoneNumber, AccessLevel access)
```
Comment (yevhenbilushko) [inline Module3/Models/User.cs]: no need

File: Module3/Models/Product.cs (hunk 1 above)
Comment (OlegMazurenko) [reply Module3/Models/Product.cs:9]: OrderId instead of Name? Why? I dont think i even need OrderId property here.

Comment (yevhenbilushko) [reply Module3/Models/Product.cs:9]: not instead 
to add
it will need for EF relation

Comment (yevhenbilushko) [reply Module3/Models/Product.cs:9]: up to u

**Review by yevhenbilushko — APPROVED** [review]

**Pull request merged**